src/lib/io/include/sol-iio.h

```diff
@@ -230,6 +230,22 @@ struct sol_str_slice
 sol_iio_read_channel_raw_buffer(struct sol_iio_channel *channel);
 
 /**
+ * @brief Perform the mount calibration.
+ *
+ * This function is meaningful only when mount_matrix is supported.
+ *
+ * @param device device to get mount matrix
+ * @param x X coordinate data to be calibrated
+ * @param y Y coordinate data to be calibrated
+ * @param z Z coordinate data to be calibrated
+ *
+ * @return true if mount_matrix is exits, false means does not support mount_matrix.
+ *
+ */
+bool
```

Comment:
Soletta APIs usually return an `int` value, where 0 is success and a negative number, failure.